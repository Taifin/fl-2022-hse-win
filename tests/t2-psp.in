$start$ = #(#$start$#)#;
$start$ = $start$$start$;
$start$ = #eps#;