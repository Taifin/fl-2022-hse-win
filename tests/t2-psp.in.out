LexToken(TERM,'$start$',1,0)
LexToken(BIND,'=',1,8)
LexToken(NTERM,'#(#',1,10)
LexToken(TERM,'$start$',1,13)
LexToken(NTERM,'#)#',1,20)
LexToken(END_OF_RULE,';',1,23)
LexToken(TERM,'$start$',2,25)
LexToken(BIND,'=',2,33)
LexToken(TERM,'$start$',2,35)
LexToken(TERM,'$start$',2,42)
LexToken(END_OF_RULE,';',2,49)
LexToken(TERM,'$start$',3,51)
LexToken(BIND,'=',3,59)
LexToken(NTERM,'#eps#',3,61)
LexToken(END_OF_RULE,';',3,66)
